source/cl/examples/MutableDispatchKHR/source/main.cpp

```diff
@@ -215,10 +215,8 @@ int main(const int argc, const char **argv) {
                                               &input_B_buffer};
       const cl_mutable_dispatch_arg_khr arg_2{2, sizeof(cl_mem),
                                               &output_buffer};
-      const cl_mutable_dispatch_arg_khr args[] = {arg_0, arg_1, arg_2};
-      const cl_mutable_dispatch_config_khr dispatch_config{
-          CL_STRUCTURE_TYPE_MUTABLE_DISPATCH_CONFIG_KHR,
-          nullptr,
+      cl_mutable_dispatch_arg_khr args[] = {arg_0, arg_1, arg_2};
+      cl_mutable_dispatch_config_khr dispatch_config{
```

Comment:
With the `clang-tidy` reports on the earlier version for variables that could be `const`, I am surprised that none are raised here. Could you check whether this really needs to be non-`const`, or whether the source file was skipped in the check?

Reply:
`clang-tidy` was being run (I manually run the `tidy-clMutableDispathcKHR` target to check) but the tool didn't seem to mandate `const` on these, but I went through and added `const` anyway